modules/ROOT/pages/update-acm.adoc

```diff
@@ -47,6 +47,24 @@ Attempting to upgrade the package without this feature enabled causes the proces
 .. Click the Delete button.
 .. Click the Confirm delete button.
 .. Continue with normal package installation.
+. If you are upgrading from a package version before 1.74, follow these steps:
+.. Create a new named credential:
+... Click *Setup* > *Named Credentials*.
+... Copy the configuration information from the named credential `anypoint_mulesoft`.
+... Create a new named credential with the name `Anypoint` and matching configuration.
+.. Update the External Data Source:
+... Click *Setup* > *External Data Source* > *Exchange* > *Edit*.
+... Set *URL* to `callout:Anypoint/odatabridge/odata.svc/`.
+... Ensure *Eligible for External Change Data Capture* is enabled.
+... Click *Save*.
+.. Update the old named credential:
+... Click *Setup* > *Named Credentials*.
+... Modify the named credential `anypoint_mulesoft`.
```

Comment:
This is a peer of the steps below it, right, and not their container?

Reply:
Correct, this is a peer of those steps and not their container.